doc/sphinx/source/ops/concat.rst

```diff
@@ -12,9 +12,8 @@ Concat
 Description
 ===========
 
-Produces a single output tensor of the same element type and shape as ``arg``,
-where the value at each coordinate of ``output`` is the absoloute value of the
-value at each ``arg`` coordinate.
+Produce from ``Nodes`` of ``args`` some outputs with the same attributes
```

Comment:
Think this edit may have been a mistake?

Reply:
However, the old docs are wrong... they appear to be just a copy-and-paste of the docs for `Abs` which was not edited... please feel free to punt on this and open a JIRA assigned to me or Scott to fix this.

Reply:
Got it, will do